### PR TITLE
A few code cleanups, exception and comparison fix

### PR DESCRIPTION
**Pull request by bang-uin**

### CODE REVIEW

File: honssh/base_auth_handler.py [hunk 1]
```diff
@@ -35,29 +35,33 @@
...
     
-    def __init__(self, server):
+    def __init__(self, server, name):
         self.server = server
+        self.name = name
         self.auth_plugin = None
         self.cfg = self.server.cfg
-        
+
+        self.connection_timeout = 10
```
Comment (tnich) [inline honssh/base_auth_handler.py]: Doesn't this need to be the connection_timeout from the config file? Rather than fixed?
Or am I missing something?

Comment (tnich) [inline honssh/base_auth_handler.py]: Of course... inheritance :smile: 
I'll just test it out and then merge it in

Comment (bang-uin) [inline honssh/base_auth_handler.py]: I'll wait for this merge before adding the new pull requests. As it may create some conflicts.

Comment (tnich) [inline honssh/base_auth_handler.py]: Good plan :)

**Comment by tnich** [comment]
Merged, thanks for the effort :D

**Comment by bang-uin** [comment]
You're welcome.

**Pull request merged**